infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/transport/sample/GRPCLogReplicationServerChannelAdapter.java

```diff
@@ -36,7 +37,8 @@ public GRPCLogReplicationServerChannelAdapter(ServerContext serverContext, LogRe
         super(serverContext, router);
         this.service = new GRPCLogReplicationServerHandler(router);
         this.port = Integer.parseInt((String) serverContext.getServerConfig().get("<port>"));
-        this.server = ServerBuilder.forPort(port).addService(service).build();
+        this.server = ServerBuilder.forPort(port).addService(service)
```

Comment:
I am fine with this change as it does look like the server will utilize this executorService.  Can you add a comment explaining that if multiple requests are received concurrently, ordering will not be guaranteed.  The default executorService uses a thread pool with multiple threads([GrpcUtils.class](https://github.com/grpc/grpc-java/blob/master/core/src/main/java/io/grpc/internal/GrpcUtil.java#L549)